security/middleware.py

```diff
@@ -24,6 +25,42 @@
     MiddlewareMixin = object
 
 
+class CustomLogoutMixin(object):
+    """
+    If the CUSTOM_LOGOUT_MODULE is set in Django config, import
+    and use that when performing a logout.
+    """
+
+    class Messages(object):
```

Comment:
This might be nicer as an `Enum`? Although i'm not sure the supported Python version of this project, as `Enum` is 3.4+

https://docs.python.org/3/library/enum.html

Reply:
The project supports Python 2.7+ so sadly no enums